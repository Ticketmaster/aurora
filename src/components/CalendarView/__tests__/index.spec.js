import React from "react";
import renderer from "react-test-renderer";
import {
  DayTile,
  DayTileDate,
  DayTileItem,
  DayTileButton,
  DayTileMoreButton,
  DayTileOverflowButton
} from "../";

describe("CalendarView", () => {
  it("should render without errors", () =>
    expect(
      renderer
        .create(
          <DayTile>
            <DayTile.Header>
              <DayTileDate accent>10</DayTileDate>
            </DayTile.Header>
            <DayTileItem.Group>
              <DayTileItem>
                <DayTileItem.Header>
                  <DayTileOverflowButton
                    aria-label="More Info"
                    onClick={() => {}}
                  />
                </DayTileItem.Header>
                <DayTileItem.Content>
                  <DayTileItem.ContentColumn>
                    <DayTileItem.Title>
                      Henderson, NV — Sunset Station Outdoor Amphitheater
                    </DayTileItem.Title>
                    <DayTileItem.SubTitle>
                      On Sale Starts Soon
                    </DayTileItem.SubTitle>
                  </DayTileItem.ContentColumn>
                </DayTileItem.Content>
                <DayTileItem.Footer>
                  <DayTileButton.Group>
                    <DayTileButton href="#">2:00 pm</DayTileButton>
                    <DayTileButton href="#">6:00 pm</DayTileButton>
                  </DayTileButton.Group>
                  <DayTileMoreButton onClick={() => {}}>
                    +4 More Times
                  </DayTileMoreButton>
                </DayTileItem.Footer>
              </DayTileItem>
              <DayTileItem>
                <DayTileItem.Header>
                  <DayTileOverflowButton
                    aria-label="More Info"
                    onClick={() => {}}
                  />
                </DayTileItem.Header>
                <DayTileItem.Content>
                  <DayTileItem.ContentColumn>
                    <DayTileItem.Title>
                      Henderson, NV — Sunset Station Outdoor Amphitheater
                    </DayTileItem.Title>
                  </DayTileItem.ContentColumn>
                </DayTileItem.Content>
                <DayTileItem.Footer>
                  <DayTileButton.Group>
                    <DayTileButton href="#">2:00 pm</DayTileButton>
                    <DayTileButton href="#">6:00 pm</DayTileButton>
                  </DayTileButton.Group>
                  <DayTileMoreButton onClick={() => {}}>
                    +4 More Times
                  </DayTileMoreButton>
                </DayTileItem.Footer>
              </DayTileItem>
            </DayTileItem.Group>
          </DayTile>
        )
        .toJSON()
    ).toMatchSnapshot());
});
