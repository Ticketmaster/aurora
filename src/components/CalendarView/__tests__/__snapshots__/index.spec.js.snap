// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarView should render without errors 1`] = `
.c1 > .ktYHxE + .c2 {
  margin-top: 8px;
}

.c0:hover .fiSZXT {
  color: #026cdf;
  fill: currentColor;
}

<div
  className="day-tile GnYRo"
>
  <header
    className="gVZTwB"
  >
    <div
      className="capNqu"
    >
      <div
        className="text text--dark text--primary day-tile-date__text day-tile-date__text--accent hPAfYZ cIRWLi"
        disabled={false}
        size="kilo"
      >
        10
      </div>
    </div>
  </header>
  <article
    className="bYjpGe"
  >
    <header
      className="bDMoiK"
    >
      <div
        className="dBDuqP"
      >
        <button
          aria-label="More Info"
          className="c0 fkIjyZ gsugBH"
          onClick={[Function]}
          role="button"
          size={30}
        >
          <svg
            className="fiSZXT"
            fill="currentColor"
            height={21}
            viewBox="0 0 5 21"
            width={5.25}
          >
            <path
              d="M0 2.5C0 1.81.242 1.22.727.732A2.394 2.394 0 0 1 2.492 0a2.42 2.42 0 0 1 1.773.732C4.755 1.221 5 1.81 5 2.5s-.245 1.28-.735 1.768A2.42 2.42 0 0 1 2.492 5C1.8 5 1.212 4.756.727 4.268A2.418 2.418 0 0 1 0 2.5zm0 8.008c0-.692.242-1.283.727-1.773A2.39 2.39 0 0 1 2.492 8c.692 0 1.283.245 1.773.735S5 9.815 5 10.508a2.39 2.39 0 0 1-.735 1.765A2.43 2.43 0 0 1 2.492 13c-.692 0-1.28-.242-1.765-.727A2.403 2.403 0 0 1 0 10.508zM0 18.5c0-.69.242-1.28.727-1.768A2.394 2.394 0 0 1 2.492 16a2.42 2.42 0 0 1 1.773.732c.49.489.735 1.078.735 1.768s-.245 1.28-.735 1.768A2.42 2.42 0 0 1 2.492 21c-.692 0-1.28-.244-1.765-.732A2.418 2.418 0 0 1 0 18.5z"
            />
          </svg>
        </button>
      </div>
    </header>
    <div
      className="kAdUdq"
    >
      <div
        className="TtAyf"
      >
        <div
          className="text text--dark text--primary bSXNxK dIqltR"
          disabled={false}
          size="uno"
        >
          Henderson, NV — Sunset Station Outdoor Amphitheater
        </div>
        <div
          className="text text--dark text--secondary bSXNxK jUhKrd"
          disabled={false}
          size="uno"
        >
          On Sale Starts Soon
        </div>
        <div
          className="text text--dark text--primary bhdbzH hGozFO"
          disabled={false}
          size="uno"
        >
          On Sale Starts Soon
        </div>
      </div>
    </div>
    <footer
      className="fZIqZA"
    >
      <div
        className="c1 cnZIRB"
      >
        <div
          className="c2 ktYHxE"
        >
          <a
            className="SWBLm"
            href="#"
            rel={undefined}
            size="small"
          >
            2:00 pm
          </a>
        </div>
        <div
          className="c2 ktYHxE"
        >
          <a
            className="SWBLm"
            href="#"
            rel={undefined}
            size="small"
          >
            6:00 pm
          </a>
        </div>
      </div>
      <div
        className="jXpsgi"
      >
        <button
          className="cJSbnz clbKAW"
          href={null}
          onClick={[Function]}
          rel="_self"
          role="button"
          size="uno"
          target=""
        >
          +4 More
        </button>
      </div>
    </footer>
  </article>
  <div
    className="giyzyZ"
  />
  <article
    className="bYjpGe"
  >
    <header
      className="bDMoiK"
    >
      <div
        className="dBDuqP"
      >
        <button
          aria-label="More Info"
          className="c0 fkIjyZ gsugBH"
          onClick={[Function]}
          role="button"
          size={30}
        >
          <svg
            className="fiSZXT"
            fill="currentColor"
            height={21}
            viewBox="0 0 5 21"
            width={5.25}
          >
            <path
              d="M0 2.5C0 1.81.242 1.22.727.732A2.394 2.394 0 0 1 2.492 0a2.42 2.42 0 0 1 1.773.732C4.755 1.221 5 1.81 5 2.5s-.245 1.28-.735 1.768A2.42 2.42 0 0 1 2.492 5C1.8 5 1.212 4.756.727 4.268A2.418 2.418 0 0 1 0 2.5zm0 8.008c0-.692.242-1.283.727-1.773A2.39 2.39 0 0 1 2.492 8c.692 0 1.283.245 1.773.735S5 9.815 5 10.508a2.39 2.39 0 0 1-.735 1.765A2.43 2.43 0 0 1 2.492 13c-.692 0-1.28-.242-1.765-.727A2.403 2.403 0 0 1 0 10.508zM0 18.5c0-.69.242-1.28.727-1.768A2.394 2.394 0 0 1 2.492 16a2.42 2.42 0 0 1 1.773.732c.49.489.735 1.078.735 1.768s-.245 1.28-.735 1.768A2.42 2.42 0 0 1 2.492 21c-.692 0-1.28-.244-1.765-.732A2.418 2.418 0 0 1 0 18.5z"
            />
          </svg>
        </button>
      </div>
    </header>
    <div
      className="kAdUdq"
    >
      <div
        className="TtAyf"
      >
        <div
          className="text text--dark text--primary bSXNxK dIqltR"
          disabled={false}
          size="uno"
        >
          Henderson, NV — Sunset Station Outdoor Amphitheater
        </div>
      </div>
    </div>
    <footer
      className="fZIqZA"
    >
      <div
        className="c1 cnZIRB"
      >
        <div
          className="c2 ktYHxE"
        >
          <a
            className="SWBLm"
            href="#"
            rel={undefined}
            size="small"
          >
            2:00 pm
          </a>
        </div>
        <div
          className="c2 ktYHxE"
        >
          <a
            className="SWBLm"
            href="#"
            rel={undefined}
            size="small"
          >
            6:00 pm
          </a>
        </div>
      </div>
      <div
        className="jXpsgi"
      >
        <button
          className="cJSbnz clbKAW"
          href={null}
          onClick={[Function]}
          rel="_self"
          role="button"
          size="uno"
          target=""
        >
          +4 More
        </button>
      </div>
    </footer>
  </article>
</div>
`;
